### ISSUE TITLE
Affiche de la carte selon la dernière position de l'utilisateur (cookies)

### ISSUE THREAD

**Issue by mmohadIGN** [ignf-sidc/diffusion_lidar#15]
Intitulé

    En tant qu’utilisateur
    Je veux qu'à l'ouverture du site la carte soit zoomé sur ma dernière position
    Afin de gagné en rapidité

Spécifications

    Un bouton loupe est présent sur la page
    au clique sur la loupe je peux saisir une adresse qui m’intéresse ou une commune
    Lorsque la recherche est validé la carte est centrée sur ma localité ( zoom ?)

Critères d'acceptation

Le ticket est terminé lorsque :

    Lorsque je peux saisir une adresse/ commune
    La carte se centre sur la zone recherché
    Une loupe indiquant la fonctionnalité est présente

Commentaires

**Comment by loux22** [comment]
voir si on doit faire accepter ou non les cookies à l'utilisateur 
voir si on le met automatiquement sans acceptation de l'utilisateur si on doit l'informer

**Comment by mmohadIGN** [comment]
On est obligé de lui demander. A minima s'il accepte l'utilisation de cookies nécessaire pour le fonctionnement du site. Et s'il accepte la géolocalisation. 
Par contre, je ne suis pas sûr des lois à ce sujet, je sais qu'on est imposé de demander. J'ai un doute sur la formulation donc à redemander à Valentin ou Terry.

**Comment by TerryMoreauIGN** [comment]
Yes c'est important / indispensable de leur demander ! Pour cette partie, il me semble que c'est du côté de Guillaume Remaud (SDPU) que ca se passe...

**Comment by TerryMoreauIGN** [comment]
Vu avec Valentin : ticket inutile à traiter pour ce POC, mais à voir les possibilités de cookies "nécessaires" côté GPF !
